cli/crates/federated-dev/src/dev/gateway_nanny.rs

```diff
@@ -57,7 +57,7 @@ pub(super) fn new_gateway(graph: Option<FederatedGraph>, config: &FederatedGraph
         EngineEnv {
             fetcher: runtime_local::NativeFetcher::runtime_fetcher(),
             cache: cache.clone(),
-            trusted_documents: runtime_noop::trusted_documents::NoopTrustedDocuments.into(),
+            trusted_documents: Box::new(runtime_noop::trusted_documents::NoopTrustedDocuments),
```

Comment:
For most runtime stuff I usually have a struct like:
```
struct TrustedDocuments(Arc<dyn TrustedDocumentsInner>)

impl Defer for TrustedDocuments {
   type Target = dyn TrustedDocumentsInner;
}
```

to hide the dynamic part

Reply:
makes it easier to manipulate elsewhere

Reply:
done